rosys/vision/camera/camera.py

```diff
@@ -26,12 +25,14 @@ def __init__(self,
                  streaming: bool = True,
                  polling_interval: float = 0.1,
                  base_path_overwrite: str | None = None,
+                 image_history_length: int = 256,
                  **kwargs) -> None:
         super().__init__(**kwargs)
         self.id: str = id
         self.name = name or self.id
         self.connect_after_init = connect_after_init
-        self.images: deque[Image] = deque(maxlen=self.MAX_IMAGES)
+        self.image_history_length = image_history_length
```

Comment:
Maybe we should make this field private, because we don't want the user to change it after instantiation. The deque wouldn't notice a change.

Reply:
Or we don't save it at all and use `deque.maxlen` in to_dict

Reply:
Great idea!